daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Relation.scala

```diff
@@ -15,38 +14,31 @@ object Relation {
   // as the same
   // this fits our purposes for the moment
   /** A Relation. */
-  type Relation[A, B] = immutable.Map[A, Set[B]]
+  @deprecated("2.4.0", "use com.daml.lf.Relation")
+  type Relation[A, B] = data.Relation[A, B] // alias for Map[A, Set[B]]
+  @deprecated("2.4.0", "use com.daml.lf.Relation")
+  val Relation = this
 
-  object Relation {
+  def empty[A, B]: data.Relation[A, B] = Map.empty
 
-    def merge[A, B](r: Relation[A, B], pair: (A, Set[B])): Relation[A, B] =
-      r.updated(pair._1, r.getOrElse(pair._1, Set.empty[B]).union(pair._2))
```

Comment:
I guess `merge` is also not used (hence why we've deleted it)?

Reply:
Yes